gramps_webapi/api/resources/tasks.py

```diff
@@ -19,14 +19,14 @@
 
 """Background task resources."""
 
-import json
+#import json
 from http import HTTPStatus
 
 from celery.result import AsyncResult
 from flask import abort
 
 from . import ProtectedResource
-
+from gramps.gen.lib.json_utils import object_to_string
```

Comment:
```suggestion

```

Reply:
Probably want to change that to `dict_to_string`